PlatoPlugin/src/org/workcraft/plugins/plato/tasks/PlatoTask.java

```diff
@@ -99,6 +100,19 @@ public PlatoTask(File inputFile, Object[] includeList, boolean fst, boolean syst
             }
         }
 
+        if (system) {
+            File hidir = new File(System.getProperty("java.io.tmpdir", null), "hidir");
+            if (!hidir.exists() && !hidir.mkdir()) {
+                try {
+                    throw new IOException("Failed to create temporary directory " + hidir);
```

Comment:
@jrbeaumont This is strange code -- trow an exception in try statement. Could you double check it is correct?